x-pack/plugins/security_solution/public/management/components/response_action_file_download_link/response_action_file_download_link.tsx

```diff
@@ -123,20 +150,16 @@ export const ResponseActionFileDownloadLink = memo<ResponseActionFileDownloadLin
           data-test-subj={getTestId('passcodeMessage')}
           className="eui-displayInline"
         >
-          <FormattedMessage
-            id="xpack.securitySolution.responseActionFileDownloadLink.passcodeInfo"
-            defaultMessage="(ZIP file passcode: {passcode})."
-            values={{
-              passcode: 'elastic',
-            }}
-          />
+          {FILE_PASSCODE_INFO_MESSAGE}
         </EuiText>
         <EuiText size={textSize} data-test-subj={getTestId('fileDeleteMessage')}>
-          <FormattedMessage
-            id="xpack.securitySolution.responseActionFileDownloadLink.deleteNotice"
-            defaultMessage="Files are periodically deleted to clear storage space. Download and save file locally if needed."
-          />
+          {FILE_DELETED_MESSAGE}
         </EuiText>
+        {isTruncatedFile && (
+          <EuiText size={textSize} data-test-subj={getTestId('fileTruncatedMessage')}>
```

Comment:
From the screen capture, the text added here really does not stand out from the other text we already display below the download link. Thinking maybe you add a spacer above it and perhaps also an `alert` icon to the left of the text? What do you think?

Reply:
done 71f831d15916fbbce08e98237ca9729d80b7f4a5

Reply:
This looks good 👍